modules/nf-core/annotsv/annotsv/main.nf

```diff
@@ -8,12 +8,11 @@ process ANNOTSV_ANNOTSV {
         'quay.io/biocontainers/annotsv:3.3.4--py311hdfd78af_1' }"
 
     input:
-    tuple val(meta), path(variants), path(variants_index)
+    tuple val(meta), path(variants), path(variants_index), path(snv_vcf)
```

Comment:
how about naming this something like `candidate_snv`. I know we have the meta.yml and all. But `variants` and `snv` in one part is confusing naming to me 🙈

Reply:
you got a point there! I'll maybe rename the original variants to 'SV_vcf` or something

Reply:
Done!